### PR TITLE
Massi starting front end

### PR DESCRIPTION
**Pull request by MassimilianoMura**

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -21,23 +21,36 @@
...
     <!-- your app's js -->
     <script src="js/app.js"></script>
+    <script src="js/routes.js"></script>
     <script src="js/detourController.js"></script>
+    <script src="js/mapController.js"></script>
     <script src="js/recommendationFactory.js"></script>
+
+    <script src='//maps.googleapis.com/maps/api/js?sensor=false'></script>
   </head>
-  <body ng-controller="DetourController as Ctrl">
```
Comment (katsuraku) [inline www/index.html]: DONT DELETE THIS?!?!?!?!

File: www/index.html [hunk 2]
```diff
@@ -21,23 +21,36 @@
...
-        <br>
-        <br>
-        <br>
-        <button class="button" ng-click="Ctrl.send()"></button>
-        <p class="AjaxResponse">
-          {{ Ctrl.response }}
-        </p>
-      </ion-content>
-    </ion-pane>
-  </body>
```
Comment (katsuraku) [inline www/index.html]: Integrate this stuff plz

File: package.json [hunk 3]
```diff
@@ -33,4 +33,4 @@
     "ios",
     "android"
   ]
-}
+}
```
Comment (katsuraku) [inline package.json]: Wtf is this?

File: www/index.html (hunk 1 above)
Comment (MassimilianoMura) [inline www/index.html]: Dear Kirsten,
Perhaps next time you might consider being a little more considerate of your teammates' feelings when adding a line note?
Kind regards, 
Massimiliano Mura

File: package.json (hunk 3 above)
Comment (MassimilianoMura) [inline package.json]: scusa??

**Pull request merged**